### PR TITLE
Improve Dynamic Topic Model (DTM) documentation

### LINKED ISSUE #2637: Incomplete DTM docs
In https://radimrehurek.com/gensim/models/wrappers/dtmmodel.html#gensim.models.wrappers.dtmmodel.DtmModel : 

```
top_chain_var (int, optional) – Hyperparameter that affects.
```

### PR DESCRIPTION
**Pull request by joelowj**
make correction for the datatype for top_chain_var from int to float & updated the documentation on the description of the variable. 

Fixes #2637.

### CODE REVIEW

File: gensim/models/wrappers/dtmmodel.py [hunk 1]
```diff
@@ -97,8 +97,8 @@ def __init__(self, dtm_path, corpus=None, time_slices=None, mode='fit', model='d
              Max em optimization iterations in LDA.
         alpha : int, optional
             Hyperparameter that affects sparsity of the document-topics for the LDA models in each timeslice.
-        top_chain_var : int, optional
-            Hyperparameter that affects.
+        top_chain_var : float, optional
+            Hyperparameter that determines how similar topics would be over multiple timeslice.
```
Comment (piskvorky) [inline gensim/models/wrappers/dtmmodel.py]: Can we be more explicit? What does this mean, what value should I pick as a user?

**Comment by joelowj** [comment]
@piskvorky, I guess this should be more succinct description of this variable? Thanks!

**Comment by piskvorky** [comment]
Yes, much better, thanks!

**Comment by joelowj** [comment]
@piskvorky, no problem at all! Anything else require on my side? Thanks.

**Comment by piskvorky** [comment]
I don't think so. @mpenkov will be with you presently, to review and merge. Thanks for your patience :)

File: gensim/models/wrappers/dtmmodel.py [hunk 2]
```diff
@@ -97,8 +97,10 @@ def __init__(self, dtm_path, corpus=None, time_slices=None, mode='fit', model='d
              Max em optimization iterations in LDA.
         alpha : int, optional
             Hyperparameter that affects sparsity of the document-topics for the LDA models in each timeslice.
-        top_chain_var : int, optional
-            Hyperparameter that affects.
+        top_chain_var : float, optional
+            This hyperparameter controls one of the key aspect of topic evolution which is the speed at which
+            these topics evolve. A smaller top_chain_var leads to similar word distributions over multiple timeslice.
```
Comment (mpenkov) [inline gensim/models/wrappers/dtmmodel.py]:
```suggestion
            these topics evolve. A smaller ``top_chain_var`` leads to similar word distributions over multiple timeslice.
```

**Comment by mpenkov** [comment]
OK, merged. Thank you for your contribution @joelowj !

**Pull request merged**